assignments/prototype-refactor.js

```diff
@@ -7,3 +7,127 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+/*
+=== GameObject ===
+* createdAt
+* name
+* dimensions (These represent the character's size in the video game)
+* destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+// function GameObject(attributes) {
+// this.createdAt = attributes.createdAt;
+// this.name = attributes.name;
+// this.dimensions = attributes.dimensions;
+// }
+//
+// GameObject.prototype.destroy =  function () {
+// return `${this.name} was removed from the game.`
+// }
+
+class GameObjects {
+  constructor(attrs) {
+    this.createdAt = attrs.createdAt;
+    this.name = attrs.name;
+    this.dimensions = attrs.dimensions;
+  }
+
+  destroy() {
+    return `${this.name} was removed from the game.`
+  }
+}
+
+class CharacterStats extends GameObjects {
```

Comment:
Just to keep confusion down you can actually use the same naming convention across all the super/constructors if you want to (you could go with attrs/props across the board if you wanted to)